src/Main.java

```diff
@@ -1,32 +1,19 @@
 public class Main {
 
-    public static String simbols = "abcdefghijklmnopqrstuvwxyzABCDIFGHIJKLMNOPQRSTUVWXYZ1234567890_";
-
 
     public static void main(String[] args) {
         try {
             validation("Alexandr_02", "aB123456789", "aB123456789");
-        } catch (WrongLoginException e) {
-            throw new WrongLoginException("Введите другой логин");
-        } catch (WrongPasswordException e) {
-            throw new WrongPasswordException("Проверьте пароль");
+        } catch (RuntimeException e) {
```

Comment:
вот смотри, сейчас ты сможешь поймать любое исключение, которое является наследником RuntimeException, но лучше всегда стараться только те исключения, которые та хочешь и ожидаешь, в данном случае это WrongLoginException и WrongPasswordException, т.е. лучше сделать multi-catch блок:
```suggestion
        } catch (WrongLoginException | WrongPasswordException e) {
```